src/main/java/com/hollingsworth/arsnouveau/common/spell/effect/EffectWindburst.java

```diff
@@ -35,22 +37,32 @@ public EffectWindburst() {
     public void onResolve(HitResult rayTraceResult, Level world, @NotNull LivingEntity shooter, SpellStats spellStats, SpellContext spellContext, SpellResolver resolver) {
         super.onResolve(rayTraceResult, world, shooter, spellStats, spellContext, resolver);
 
+        var loc = rayTraceResult.getLocation();
+        double x = loc.x;
+        double y = loc.y;
+        double z = loc.z;
+
+        var dummyWindCharge = new WindCharge(world, x, y, z, Vec3.ZERO);
+        if (spellStats.isSensitive()) {
+            dummyWindCharge.setOwner(shooter);
+        }
+
         world.explode(
-                        spellStats.isSensitive() ? shooter : null,
-                        null,
+                dummyWindCharge,
```

Comment:
the sensitive check should be readded as this lets the player ignore the knockback